src/frontend/src/lib/components/core/Menu.svelte

```diff
@@ -47,10 +47,12 @@
 		{/if}
 
 		{#if !settingsRoute}
-			<ButtonMenu ariaLabel={$i18n.navigation.alt.more_settings} on:click={gotoSettings}>
-				<IconSettings />
+			<!-- If the URL is passed as href instead of click event, the pages refreshes. -->
+			<!-- After a few tries, it seems it is a behavior that happens only inside the popover. -->
+			<Link href={null} ariaLabel={$i18n.navigation.alt.more_settings} on:click={gotoSettings}>
```

Comment:
That's an invalid HTML behavior I think.

From the HTML 5 [standard](https://html.spec.whatwg.org/#the-a-element)

> If the [a](https://html.spec.whatwg.org/#the-a-element) element has no [href](https://html.spec.whatwg.org/#attr-hyperlink-href) attribute, then the element [represents](https://html.spec.whatwg.org/#represents) a placeholder for where a link might otherwise have been placed, if it had been relevant, consisting of just the element's contents.